### ISSUE TITLE
Scrolling feels akwardly slow

### ISSUE THREAD

**Issue by BK-TN** [gamejolt/next-issue-tracker#125]
Using Firefox on a Win 7 machine.

Scrolling on any page feels a lot slower than on any other website. So far I haven't found any browser feature that fixes it.
I've tested the difference in pixels scrolled with a single 'tick' of the scrollwheel.. On other sites, a tick moves the page 114 pixels - on GJ Next, the page only moves 57 pixels. Same amount on the chat sidebar. The chat window itself only moves 48 pixels.

**Comment by acidpoison** [comment]
I've checked on multiple machines, and scrolling feels the same. Maybe it's your browser settings?

**Comment by hworld** [comment]
How about Chrome or IE? And is Firefox doing the smooth scrolling thing that it does?

**Comment by indiegamehunt** [comment]
This is an optimization issue as well, is it not?

**Comment by hworld** [comment]
Yeah. It is. I haven't yet looked into all perf issues. Although may as well keep this one around so I can take a look later. Any more info that I can get on it would be great though, @BK-TN

**Comment by BK-TN** [comment]
Hello again, sorry for completely forgetting about this.
- This is *not* something I see often. I've maybe encountered it once or twice on very JS-heavy pages that mess with scrolling behaviour, but unfortunately I don't have any examples.
- This is a firefox-only bug. Internet Explorer and Chrome both seem to have different scroll distances, but they are both more than twice as big as the one in Firefox.
- Turning off smooth scrolling or autoscrolling does not fix the issue.

Any other settings I can tweak to see if it helps? I'll also try running it on a different PC tomorrow.

**Comment by hworld** [comment]
Hmm, I'm not sure. It's kind of weird. It could be firefox scrolling slower in overflow: scroll containers... I actually don't touch the scrolling at all by adding any fake scroll stuff.

Does this page scroll any faster? http://wttf.gamejolt.com/terms

**Comment by BK-TN** [comment]
@hworld nope :(

**Comment by BK-TN** [comment]
Tested on a different PC now, a laptop from Acer with a way weaker CPU. Same deal, exact same scrolling distance. The page scrolls even slower when using a specific mouse (Which works fine on my desktop PC - so this might be performance related). It's still a firefox only issue.

**Comment by hworld** [comment]
Yo, hate to bring this up so late, but is it working now @BK-TN ? I changed the shell around a bit to try to get scrolling faster. But it may be unrelated to the changes I did. If so, it would be good to know.
